src/viking.js

```diff
@@ -1,11 +1,124 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function (damage) {
+  this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+
+Viking.prototype = Object.create(Soldier.prototype);
```

Comment:
I think now you get this point :)
But if you still have questions please let us know